### PR TITLE
About pages

### PR DESCRIPTION
**Pull request by mscroggs**

### CODE REVIEW

File: gathering/uk/about/index.md [hunk 1]
```diff
@@ -47,45 +19,12 @@ So here are some examples - don't expect them all to have clear answers, or even
...
-
-Katie Steckles is a mathematician based in Manchester, who gives talks and workshops on different areas of maths. She finished her PhD in 2011, and since then has talked about maths in schools, at science festivals, on BBC radio, at music festivals, as part of theatre shows and on the internet. She enjoys doing puzzles, solving the Rubik's cube and baking things shaped like maths.
-
-[Katie's website](http://www.katiesteckles.co.uk)
-
-### Rob Eastaway
-<img src="Rob.jpg" class="team-pic">
-
-Rob Eastaway is an author who is active in the popularisation of mathematics. He has a degree in Engineering and Management Science from the University of Cambridge, and was made President of the UK Mathematical Association for 2007/2008. He is also a former puzzle-writer for the New Scientist Magazine. Rob is the director of Maths Inspiration, a national programme of maths lectures for teenagers which involves some of the UK's leading maths speakers.
+## Testimonials
```
Comment (mscroggs) [inline gathering/uk/about/index.md:22]: The website had a long list of testimonials from 2010. It feels like there were too many, but perhaps having none of them is too few

File: gathering/uk/about/index.md [hunk 2]
```diff
@@ -47,45 +19,12 @@ So here are some examples - don't expect them all to have clear answers, or even
...
+If you want to read more about what MathsJam Gathering is like, you can:
 
-[Rob's website](http://www.robeastaway.com/)
+- Watch [this 5 minute MathsJam trailer](https://www.youtu.be/ZM465bN_IF8)
+- Read [this article that Peter Rowlett wrote for MSOR Connections about MathJam 2015](https://journals.gre.ac.uk/index.php/msor/article/view/278)
+- Read [Rachel Wright's thoughts about attending MathsJam in 2014 as a non-mathematician](http://www.solipsys.co.uk/new/ANonMathematicianAtMathsJam.html?TW_20160514)
```
Comment (mscroggs) [inline gathering/uk/about/index.md:27]: Describing someone as a non-mathematician is (a) probably wrong if they're attending mathsjam, and (b) a bit gatekeeper-y

File: gathering/uk/about/index.md [hunk 3]
```diff
@@ -47,45 +19,12 @@ So here are some examples - don't expect them all to have clear answers, or even
...
 
-[Maths Inspiration](http://www.mathsinspiration.com)
+## Sign up to the mailing list
+If you want us to email you when tickets for MathsJam Gathering go on sale, you can [join our mailing list](TODO: LINK HERE).
```
Comment (mscroggs) [inline gathering/uk/about/index.md:30]: mailing list link needed

File: gathering/uk/about/the-team.md [hunk 4]
```diff
@@ -3,3 +3,37 @@ layout: gathering
...
 ---
 
+MathsJam Gathering is a fully volunteer-run event, and many people play an important role in making the event a success. The people listed on this page form the core orgnisational team and lead the organisation of the event.
+
+### Colin Wright
+<img src="Colin.jpg" class="team-pic"></img>
+
+Colin has a Ph.D. from Cambridge and has worked as a research mathematician, and done research in computing. His day job is developing systems to do the maritime version of Air Traffic Control, and in his "free time" he travels around the world giving lectures on how and why maths can be interesting, fun, useful and even on occasion, exciting.
+
+### Sam Hartburn
```
Comment (mscroggs) [inline gathering/uk/about/the-team.md:13]: Need to fill in this section for everyone, and add whoever I've missed including

File: gathering/uk/about/index.md (hunk 3 above)
Comment (katiesteckles) [reply gathering/uk/about/index.md:30]: https://docs.google.com/forms/d/e/1FAIpQLSceb7PpP3qU8OcrCu_g8vHV9DVhDxo85pkP0Cqw7W4Flnkh9A/viewform?usp=sf_link

File: gathering/uk/about/index.md (hunk 2 above)
Comment (SamHartburn) [reply gathering/uk/about/index.md:27]: Agreed. Just 'Rachel Wright's thoughts about attending MathsJam in 2014' would be fine.

**Pull request merged**